src/XMLHttpRequest/XMLHttpRequest/createXMLHttpRequestOverride.ts

```diff
@@ -192,9 +192,9 @@ export const createXMLHttpRequestOverride = (
       } else {
         this.url = url
         this.method = method
-        this.async = !!async
-        this.user = user || ''
-        this.password = password || ''
+        this.async = undefined
```

Comment:
We need to be careful for the `.open()` method to have `async` as `true` by default:

https://github.com/mswjs/node-request-interceptor/blob/17453525dd439b879c428d188c05cdb37f4a1ec4/src/XMLHttpRequest/XMLHttpRequest/createXMLHttpRequestOverride.ts#L176-L182

Source: [XMLHttpRequest/open (MDN)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/open).

Reply:
Updated!